hydrography-approach/run-hydrography-pipeline.py

```diff
@@ -35,11 +44,11 @@ def main():
     state_name = "Kentucky"
 
     # Load config file
-    print("\nLoading the config file.")
+    logging.info("Loading the config file.")
```

Comment:
@varun-andhra-mapup Looks like you did not provide a response to AI comments. Please do before asking me to merge thie